src/styles/App.css

```diff
@@ -0,0 +1,117 @@
+.weather-app {
+  display: flex;
+  flex-direction: column;
+  justify-content: flex-start;
+  align-items: center;
+  min-height: 100vh;
+  padding: 2rem;
+  box-sizing: border-box;
+  color: azure;
+}
+
+.forecast {
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: center;
+  align-items: flex-start;
+  margin-top: 2rem;
+  width: 100%;
+  box-sizing: border-box;
+}
+
+.forecast-container {
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: center;
+  align-items: center;
+}
+
+.forecast-panels {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  height: 100%;
```

Comment:
The `justify-content` and `height` properties here arent doing anything, should take these out